src/EnergyPlus/SimulationManager.cc

```diff
@@ -2256,12 +2256,13 @@ namespace SimulationManager {
                                      "been retrieved.");
                     state.dataSimulationManager->WarningOut = false;
                 }
-                ShowWarningError(
+                ShowSevereError(
                     state,
                     format("Potential Node Connection Error for object {}, name={}", CType, state.dataBranchNodeConnections->CompSets(Count).CName));
                 ShowContinueError(state, "  Node Types are still UNDEFINED -- See Branch/Node Details file for further information");
                 ShowContinueError(state, format("  Inlet Node : {}", state.dataBranchNodeConnections->CompSets(Count).InletNodeName));
                 ShowContinueError(state, format("  Outlet Node: {}", state.dataBranchNodeConnections->CompSets(Count).OutletNodeName));
+                ShowFatalError(state, "Please correct either the branch nodes or the component nodes so that they match.");
```

Comment:
@RKStrand When I test the original file (with missing nodes) the new code crashes. I'm not sure why/how, but following this in the debugger, `ShowFatalError` is being called multiple times. Adding a local error flag and moving the ShowFata to the end of `ReportLoopConnections` seems to fix this.

Reply:
Weird.  Not sure what happened there.  I'll implement the recommended change and update the unit test.  Look for the mods after the holiday weekend (hopefully).  Thanks!

Reply:
@mjwitte Latest commit fixes this problem.  Avoids a hard crash as per your suggestion.  Unit test also updated.